util/reggen/uvm_reg.sv.tpl

```diff
@@ -209,13 +209,13 @@ package ${block.name}_ral_pkg;
       // create sub blocks and add their maps
 % endif
 % for b in block.blocks:
-      ${b.name} = ${gen_dv.bcname(b)}::type_id::create("${b.name}");
-      ${b.name}.configure(.parent(this));
-      ${b.name}.build(.base_addr(base_addr + ${gen_dv.sv_base_addr(b)}), .csr_excl(csr_excl));
-      ${b.name}.set_hdl_path_root("tb.dut.top_earlgrey.u_${b.name}", "BkdrRegPathRtl");
-      ${b.name}.set_hdl_path_root("tb.dut.top_earlgrey.u_${b.name}", "BkdrRegPathRtlCommitted");
-      ${b.name}.set_hdl_path_root("tb.dut.top_earlgrey.u_${b.name}", "BkdrRegPathRtlShadow");
-      default_map.add_submap(.child_map(${b.name}.default_map),
+      ${b.instance} = ${gen_dv.bcname(b)}::type_id::create("${b.instance}");
+      ${b.instance}.configure(.parent(this));
+      ${b.instance}.build(.base_addr(base_addr + ${gen_dv.sv_base_addr(b)}), .csr_excl(csr_excl));
+      ${b.instance}.set_hdl_path_root("tb.dut.top_earlgrey.u_${b.instance}", "BkdrRegPathRtl");
+      ${b.instance}.set_hdl_path_root("tb.dut.top_earlgrey.u_${b.instance}", "BkdrRegPathRtlCommitted");
+      ${b.instance}.set_hdl_path_root("tb.dut.top_earlgrey.u_${b.instance}", "BkdrRegPathRtlShadow");
+      default_map.add_submap(.child_map(${b.instance}.default_map),
```

Comment:
Except for replacing `name` attribute with `inst_name`, all other changes can be reverted.